src/me/remie/vulcan/leaguetasks/utils/LeaguePanel.java

```diff
@@ -0,0 +1,119 @@
+                    ctx.log("Something went wrong choosing tier option");
+                    return false;
+                }
+            }
+        }
+
+        final SimpleWidget typeDropDownMenu = ctx.widgets.getWidget(657,28);
+        if (typeDropDownMenu != null) {
+            if (!isOptionSelected(typeDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = typeDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 36);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing type option");
+                    return false;
+                }
+            }
+        }
+
+        final SimpleWidget areaDropDownMenu = ctx.widgets.getWidget(657,29);
+        if (areaDropDownMenu != null) {
+            if (!isOptionSelected(areaDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = areaDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 37);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing area option");
+                    return false;
+                }
+            }
+        }
+
+        final SimpleWidget completedDropDownMenu = ctx.widgets.getWidget(657,31);
+        if (completedDropDownMenu != null) {
+            if (!isOptionSelected(completedDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = completedDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 39);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing completed option");
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    private boolean chooseOptionInDropdown(SimpleWidget dropDownWidget, SimpleWidget dropDownOptionList, int option) {
+        if (dropDownWidget == null || dropDownWidget.getChild(option) == null) return false;
+        dropDownWidget.click(1);
+        ctx.sleep(1300);
+        if (dropDownOptionList != null && !dropDownOptionList.isHidden()) {
+            final SimpleWidget selectedOption = dropDownOptionList.getChild(option);
+            if (selectedOption != null) {
+                selectedOption.click(1);
+                ctx.sleep(1300);
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public boolean isLeaguePanelOpen() {
+        final SimpleWidget leaguePanelWidget = ctx.widgets.getWidget(657, 18);
+        return leaguePanelWidget != null && !leaguePanelWidget.isHidden();
+    }
+
+    public boolean openLeaguePanel() {
+        if (isLeaguePanelOpen()) return true;
```

Comment:
```suggestion
        if (isLeaguePanelOpen()) { 
            return true;
        }
```
Nitpick, please do this for all if statements above.